services/api/src/utils/logging.js

```diff
@@ -11,31 +14,63 @@ function getTracerContext() {
   return currentSpan.context();
 }
 
+function getPrettyPrintConfig() {
+  if (config.get('ENV_NAME') === 'development') {
```

Comment:
also here bedrockio/instrumentation latest version doesnt use pino for local development, which removes the need to have pino-prettier or to fight it. 

Just need to ported over the things that make sense to port

Reply:
oh I see... I didn't realize it was still in flux... but we want `pino` or something like it in development no? The log levels still make sense